### PR TITLE
Adding Form

### PR DESCRIPTION
**Pull request by sidko**
Adding details of Form and Form Points earned on PT-ezETH token.

### CODE REVIEW

File: protocols/form/config.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Form",
+  "icon": "logo.png",
+  "metadata": {
+    "pt": [
+      {
+        "chainId": 1,
+        "address": "0xf7906F274c174A52d444175729E3fa98f9bde285"
```
Comment (manh-pendle) [inline protocols/form/config.json:8]: need `,` at the end

File: protocols/form/config.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+        "integrationUrl": "https://meditations.form.network"
```
Comment (manh-pendle) [inline protocols/form/config.json:9]: need `,` at the end

**Review by manh-pendle — COMMENTED** [review]
Please reformat your `config.json` file.

File: protocols/form/config.json (hunk 1 above)
Comment (sidko) [reply protocols/form/config.json]: My bad, fixed!

File: protocols/form/config.json (hunk 2 above)
Comment (sidko) [reply protocols/form/config.json]: My bad, fixed!

**Comment by sidko** [comment]
@manh-pendle my bad, fixed the formatting!

**Pull request merged**